pkg/ansible/generate.go

```diff
@@ -91,7 +91,7 @@ openshift_master_bootstrap_enabled: True
 
 openshift_hosted_router_wait: False
 openshift_hosted_registry_wait: False
-openshift_hosted_registry_replicas: 1
+openshift_hosted_registry_replicas:  [[ .InfraSize ]]
```

Comment:
Can we tie this approach into the router we just hard coded to 1?

Reply:
definitely more useful that just determining the number of registry pods to run.